lib/generators/rolemodel/css/icons/icons_generator.rb

```diff
@@ -3,18 +3,120 @@ module Css
     class IconsGenerator < Rails::Generators::Base
       source_root File.expand_path('templates', __dir__)
 
-      def add_material_icons
+      def install_css_base
+        generate 'rolemodel:css:base' unless File.exists?(Rails.root.join('app/javascript/packs/stylesheet.js'))
+      end
+
+      def install_packages
+        run 'yarn add svgo'
         run 'yarn add material-icons'
+        run 'yarn install'
       end
 
       def add_view_helper
         copy_file 'app/helpers/icon_helper.rb'
       end
 
+      def add_webpack_loaders
+        copy_file 'config/webpack/loaders/custom-icon-loader.js'
+        copy_file 'config/webpack/loaders/svg-color-override-loader.js'
+      end
+
       def add_css
         copy_file 'app/javascript/stylesheets/components/icon.scss'
         append_file 'app/javascript/packs/stylesheets.scss', "@import 'stylesheets/components/icon';"
       end
+
+      def add_image_directories
+        run 'mkdir app/javascript/images'
+        run 'mkdir app/javascript/images/icons'
```

Comment:
Should we use `-p` and simplify to a single line?

```suggestion
        run 'mkdir -p app/javascript/images/icons'
```

Or use Thor?
```suggestion
        empty_directory 'app/javascript/images/icons'
```
Or should we just use Ruby?
```suggestion
        FileUtils.mkdir_p Rails.root.join('app/javascript/images/icons')
```

Reply:
I like the third one!